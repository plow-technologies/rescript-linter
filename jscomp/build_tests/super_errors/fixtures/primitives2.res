/* got int, wanted string */
2 ++ " things"
