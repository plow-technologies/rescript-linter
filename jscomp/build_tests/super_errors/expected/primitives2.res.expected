
  [1;31mWe've found a bug for you![0m
  [36m/.../fixtures/primitives2.res[0m:[2m2:1[0m

  1 [2mâ”‚[0m /* got int, wanted string */
  [1;31m2[0m [2mâ”‚[0m [1;31m2[0m ++ " things"
  3 [2mâ”‚[0m 

  This has type: [1;31mint[0m
  But this function argument is expecting: [1;33mstring[0m
  
  You can convert [1;33mint[0m to [1;33mstring[0m with [1;33mBelt.Int.toString[0m.